backend/app/app/urls.py

```diff
@@ -1,22 +1,11 @@
-"""
-URL configuration for app project.
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/4.2/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
 from django.contrib import admin
 from django.urls import path
+from polls.views import register_user_view, login_user_view, error_page_view, series_page_view
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
+    path('admin/', admin.site.urls),
+    path('api/v1/auth/login/', login_user_view, name='login'),
+    path('api/v1/auth/register/', register_user_view, name='register'),
+    path('api/v1/auth/series_page/', series_page_view, name='series_page'),
+    path('error/', error_page_view, name='error_page'),
```

Comment:
We don't need this URL since each endpoint will return errors related to endpoint logic specifically themself